package.json

```diff
@@ -36,32 +35,40 @@
       "afterEach",
       "beforeEach",
       "describe",
-      "it"
+      "it",
+      "expect"
```

Comment:
Aren't we `require`ing `expect.js` in our tests?

Reply:
It's entirely possible this is no longer required (I went thought a lot of different options before locking in on this)  Try removing it and see how it runs.